src/libraries/System.IO.FileSystem/tests/Directory/Exists.cs

```diff
@@ -224,8 +224,7 @@ public void WindowsWhiteSpaceAsPath_ReturnsFalse(string component)
 
         }
 
-        [Fact]
-        [PlatformSpecific(CaseInsensitivePlatforms)]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsCaseInsensitiveOS))]
```

Comment:
just for my education: what is the benefit of doing that?

Reply:
As we had 2 ways to check "case sensitivity" having only one will make it it less confusing. And I kept boolean parameter because in some places only bool can be used.